assets/src/edit-story/components/library/test/text/textTab.js

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { act, fireEvent } from '@testing-library/react';
+
+/**
+ * Internal dependencies
+ */
+import { FontProvider } from '../../../../app/font';
+import { TextTab } from '../../panes/text';
+import APIContext from '../../../../app/api/context';
+import { DEFAULT_PRESET } from '../../panes/text/textPresets';
+import { renderWithTheme } from '../../../../testUtils/index';
+
+jest.mock('../../useLibrary');
+import useLibrary from '../../useLibrary';
+
+describe('TextTab', () => {
+  const insertElement = jest.fn();
+  beforeAll(() => {
+    useLibrary.mockImplementation(() => ({
+      actions: {
+        insertElement: insertElement,
+      },
+    }));
+  });
+
+  it('should insert text with default text style on shortcut click', async () => {
+    const getAllFontsPromise = Promise.resolve([]);
+    const apiContextValue = {
+      actions: {
+        getAllFonts: () => getAllFontsPromise,
+      },
+    };
+    await act(async () => {
+      const { getByLabelText } = renderWithTheme(
+        <APIContext.Provider value={apiContextValue}>
+          <FontProvider apiContextValue>
+            <TextTab />
+          </FontProvider>
+        </APIContext.Provider>
+      );
+
+      await getAllFontsPromise;
+
+      fireEvent.click(getByLabelText('Add new text element'));
+    });
+
+    expect(insertElement).toHaveBeenCalledTimes(1);
+    expect(insertElement).toHaveBeenCalledWith('text', {
+      ...DEFAULT_PRESET,
+      width: 160,
```

Comment:
Q: Shouldn't the width here be the same as when added via QuickAction or the shortcut link?

Reply:
I decided to leave the width here as the same because in the issue we were going to move changing the textbox width and position out of the scope of this issue.

Would you like me to change the width to be consistent across both of them? Which one should I use? 

I'm not sure what width to use (the issue says 360px but that seems wrong considering in the Figma specs [text > write specs > recommended text width] the textbox width is capped at 332px. Both half the width of the page and 160px seems fine to me. @samitron7  thoughts?

Reply:
It just seems that it would make sense to have these consistent, that's all, choosing any value should be OK if this will be addressed separately later anyway to finalize. Thoughts?

Reply:
Sounds good. I chose the 140 px value from TextTab since it better spans the default text. I then moved the default element width value to the presets and added them directly there instead. PTAL.